addons/overpressure/initSettings.inc.sqf

```diff
@@ -1,7 +1,15 @@
 private _category = [LELSTRING(common,categoryUncategorized), LLSTRING(DisplayName)];
 
 [
-    QGVAR(distanceCoefficient), "SLIDER",
+    QGVAR(backblastDistanceCoefficient), "SLIDER",
+    [ELSTRING(statBackblastRange,Overpressure), LSTRING(distanceCoefficient_toolTip)],
```

Comment:
`ELSTRING(statBackblastRange,Overpressure)` makes no sense. Do you mean `LSTRING(statBackblastRange)`?

In either case, the chosen stringtable entry is not suited for this, a new one needs be created.

Reply:
> `ELSTRING(statBackblastRange,Overpressure)` makes no sense. Do you mean `LSTRING(statBackblastRange)`?
> 
> In either case, the chosen stringtable entry is not suited for this, a new one needs be created.

Aye, fixing.